patch/atmega328p.yaml

```diff
@@ -1,3 +1,6 @@
 _include:
-  - "common/wdt.yaml"
   - "common/ac.yaml"
+  - "common/timer/tc0.yaml"
+  - "common/twi.yaml"
+  - "common/usart.yaml"
+  - "common/wdt.yaml"
```

Comment:
Those changes were already merged in bfbd5bc006ad745037c004d90405ac216cb64509.  Maybe you need to rebase your branch ontop of my current master?

Reply:
Yeah I think Github is glitching out here, I wasn't sure why it was showing up.  Will try rebasing.